source/_patterns/basics/forms/form-group.hbs

```diff
@@ -0,0 +1,47 @@
+{{!--
+formRightAligned - when set to true it'll display the label on the right of the form
+formHelp - when set to true it'll display a help text line
+--}}
+
+{{#if formRightAligned}}
+  <div class="form-group row {{ formGroupModifierClass }}">
+    <label class="form-control-label col-sm-2 col-form-label" for="{{formInputId}}">{{ formLabel }}</label>
+
+    <div class="col-sm-10">
+      {{#if formTextarea}}
+        <textarea class="form-control" id="{{ formInputId }}" rows="3"></textarea>
+      {{else if formSelect }}
+        {{> basics-select }}
+      {{else}}
+        {{> basics-input }}
+      {{/if}}
+
+      {{#if formHelp }}
+        <small class="form-text text-muted" id="{{formHelpId}}">{{ formHelpText }}</small>
+      {{/if}}
+    </div>
```

Comment:
Following up on my previous comment, wrap this `</div>` (line 22) with another `{{#if formRightAligned}}`